db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/OperatorToString.java

```diff
@@ -356,6 +356,9 @@ else if (operand instanceof SimpleStringOperatorNode) {
                     SimpleStringOperatorNode sso = (SimpleStringOperatorNode) operand;
                     return format("%s(%s) ", operatorString, opToString2(sso.getOperand(), vars));
                 }
+                else if (operand instanceof NotNode) {
```

Comment:
i think this is effectively the same code as lines 355-357. SimpleStringOperatorNode doesn't override getOperand, and even if - if i understand java inheritance correctly - i think uop.getOperand() would call the same as (((SimpleStringOperatorNode)uop).getOperand() .
so i think lines 355-361 can be replaced by
```
else if (operand instanceof SimpleStringOperatorNode || operand instanceof NotNode) {
    return format("%s(%s) ", operatorString, opToString2(uop.getOperand(), vars));
}
```

Reply:
Good eye, I applied that change.